src/dataStore/DripsDataStore.sol

```diff
@@ -0,0 +1,240 @@
+    }
+
+    /// @notice Store a list of splits receivers.
+    /// Storing an already stored list is valid, and it doesn't do anything.
+    /// @param splits The splits receivers list to store.
+    /// @return hash The hash of the stored list, that can be used to load it.
+    /// If `splits` is empty, it's `EMPTY_HASH`, which is `0`.
+    /// It's always equal to the splits hash in `Drips` of the stored list.
+    function storeSplits(SplitsReceiver[] calldata splits) external returns (bytes32 hash) {
+        if (splits.length == 0) return EMPTY_HASH;
+        bytes memory data = abi.encode(splits);
+        hash = keccak256(data);
+        if (_splitsPointers[hash] == address(0)) {
+            _splitsPointers[hash] = _store(data);
+            emit SplitsStored(hash);
+        }
+    }
+
+    /// @notice Checks if the list of splits receivers is stored.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return isStored True if the list of splits receivers is stored.
+    /// If `hash` is `EMPTY_HASH`, which is `0`, always returns true.
+    function isSplitsStored(bytes32 hash) public view returns (bool isStored) {
+        if (hash == EMPTY_HASH) return true;
+        return _splitsPointers[hash] != address(0);
+    }
+
+    /// @notice Loads the list of splits receivers.
+    /// Reverts if the list with the given hash isn't stored, see `isSplitsStored`.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return splits The loaded list of splits receivers.
+    function loadSplits(bytes32 hash) external view returns (SplitsReceiver[] memory splits) {
+        if (hash == EMPTY_HASH) return splits;
+        _loadAndReturn(_splitsPointers[hash]);
+    }
+
+    /// @notice Store a list of account metadata.
+    /// Storing an already stored list is valid, and it doesn't do anything.
+    /// @param accountMetadata The account metadata list to store.
+    /// @return hash The hash of the stored list, that can be used to load it.
+    /// If `splits` is empty, it's `EMPTY_HASH`, which is `0`.
+    function storeAccountMetadata(AccountMetadata[] calldata accountMetadata)
+        external
+        returns (bytes32 hash)
+    {
+        if (accountMetadata.length == 0) return EMPTY_HASH;
+        bytes memory data = abi.encode(accountMetadata);
+        hash = keccak256(data);
+        if (_accountMetadataPointers[hash] == address(0)) {
+            _accountMetadataPointers[hash] = _store(data);
+            emit AccountMetadataStored(hash);
+        }
+    }
+
+    /// @notice Checks if the list of account metadata is stored.
+    /// @param hash The hash of the list, see `storeSplits`.
+    /// @return isStored True if the list of account metadata is stored.
+    /// If `hash` is `EMPTY_HASH`, which is `0`, always returns true.
+    function isAccountMetadataStored(bytes32 hash) public view returns (bool isStored) {
+        if (hash == EMPTY_HASH) return true;
```

Comment:
```suggestion
        if (hash == EMPTY_HASH) return false;
```
I think it should be `false` here. It would work without the `if` as well.

Reply:
Returning `true` matches the behavior of `loadAccountMetadata`, which is not throwing and returning the requested data, which is an empty array. It also matches `storeAccountMetadata`, if you pass into it an empty array and get back a hash (in this case `EMPTY_HASH`), then you expect that `isAccountMetadataStored` will return for that hash `true`.